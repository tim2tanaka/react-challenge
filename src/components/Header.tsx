import { HeaderProps } from '../types';

export function Header({ title }: HeaderProps) {
  return (
    <header className="app-header">
      <h2>{title}</h2>
    </header>
  );
}
