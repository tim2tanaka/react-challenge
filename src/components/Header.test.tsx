import { render } from '@testing-library/react';
import { Header } from './Header';

describe('Header', () => {
  test('render Header', () => {
    const { container } = render(<Header title="Test Counties App" />);
    expect(container).toMatchSnapshot();
  });
});
