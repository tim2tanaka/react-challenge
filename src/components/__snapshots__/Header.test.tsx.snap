// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render Header 1`] = `
<div>
  <header
    class="app-header"
  >
    <h2>
      Test Counties App
    </h2>
  </header>
</div>
`;
